ale_linters/elm/make.vim

```diff
@@ -57,23 +38,142 @@ function! ale_linters#elm#make#Handle(buffer, lines) abort
     return l:output
 endfunction
 
+function! ale_linters#elm#make#HandleElm019Line(line, output) abort
+    let l:report = json_decode(a:line)
+
+    if l:report.type is? 'error'
+        " General problem
+        let l:details = ale_linters#elm#make#ParseMessage(l:report.message)
+
+        if empty(l:report.path)
+            let l:report.path = 'Elm'
+        endif
+
+        if ale_linters#elm#make#FileIsBuffer(l:report.path)
+            call add(a:output, {
+                        \    'lnum': 1,
+                        \    'type': 'E',
+                        \    'text': l:details,
+                        \})
+        else
+            call add(a:output, {
+                        \    'lnum': 1,
+                        \    'type': 'E',
+                        \    'text': l:report.path .' - '. l:details,
+                        \    'detail': l:report.path ." ----------\n\n". l:details,
+                        \})
+        endif
+    else
+        " Compilation errors
+        for l:error in l:report.errors
+            let l:file_is_buffer = ale_linters#elm#make#FileIsBuffer(l:error.path)
+
+            for l:problem in l:error.problems
+                let l:details = ale_linters#elm#make#ParseMessage(l:problem.message)
+
+                if l:file_is_buffer
+                    " Buffer module has problems
+                    call add(a:output, {
+                                \    'lnum': l:problem.region.start.line,
+                                \    'col': l:problem.region.start.column,
+                                \    'end_lnum': l:problem.region.end.line,
+                                \    'end_col': l:problem.region.end.column,
+                                \    'type': 'E',
+                                \    'text': l:details,
+                                \})
+                else
+                    " Imported module has problems
+                    let l:location = l:error.path .':'. l:problem.region.start.line
+                    call add(a:output, {
+                                \    'lnum': 1,
+                                \    'type': 'E',
+                                \    'text': l:location .' - '. l:details,
+                                \    'detail': l:location ." ----------\n\n". l:details,
+                                \})
+                endif
+            endfor
+        endfor
+    endif
+endfunction
+
+function! ale_linters#elm#make#HandleElm018Line(line, output) abort
+    let l:errors = json_decode(a:line)
+
+    for l:error in l:errors
+        let l:file_is_buffer = ale_linters#elm#make#FileIsBuffer(l:error.file)
+
+        if l:file_is_buffer
+            " Current buffer has problems
+            call add(a:output, {
+                        \    'lnum': l:error.region.start.line,
```

Comment:
To make lines shorter, put `\` at the same level of indentation, and just up-indent extra lines with one level of indentation. See the rest of the codebase for examples. This applies throughout all files.